backend/config/secrets_example.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Rename this file to secrets.js so that it will be gitignored and found by server.js, then enter the information below.
```

Comment:
[nit] Change language to "Duplicate and rename" so that people don't remove `secrets_example.js` everytime

Reply:
Fixed 👍